### PR TITLE
uploading assignment8 Mi-10-MEAN/Pratiksha/html/job portal/Assignment…

### PR DESCRIPTION
**Pull request by pratikshaghule**
…8.html

### CODE REVIEW

**Review by mayank30041995 — APPROVED** [review]

File: Pratiksha/html/jobportal/Assignment8.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Assignment8</title>
+    <link rel="stylesheet" href="Assignment8.css" />
+  </head>
+  <body>
+    <div>
+      <div class="cls_wrapper">
+        <p>
```
Comment (VIP5614) [inline Pratiksha/html/jobportal/Assignment8.html:13]: aap ne p tag ke andar kyu img tag ko dala hau

Comment (VIP5614) [reply Pratiksha/html/jobportal/Assignment8.html:13]: ye tarika galat hai Sir

File: Pratiksha/html/jobportal/Assignment8.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          molestiae. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Cumque exercitationem quidem doloremque sit reprehenderit ullam
+          perferendis laudantium omnis cupiditate saepe, culpa nemo? Deserunt
+          natus velit explicabo hic vero eos modi optio debitis, dolores iure
+          labore quia at illum vitae ipsam amet ipsa alias impedit tenetur cum
+          quidem dolorem nulla! Molestias.
+        </p>
+      </div>
+      <div class="cls_wrapper1">
+        <p>
```
Comment (VIP5614) [inline Pratiksha/html/jobportal/Assignment8.html:49]: same problem

**Review by VIP5614 — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

File: Pratiksha/html/jobportal/Assignment8.html (hunk 1 above)
Comment (Pawan1005) [reply Pratiksha/html/jobportal/Assignment8.html:13]: pratiksha remove this and push file again

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request merged**